types/kap-plugin/index.d.ts

```diff
@@ -0,0 +1,45 @@
+// Type definitions for kap-plugin 1.0
+// Project: https://github.com/wulkano/kap/blob/master/docs/plugins.md
+// Definitions by: Connor Peet <https://github.com/connor4312>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Minimum TypeScript Version: 3.0
+
+import * as got from 'got';
+import { JSONSchema7 } from 'json-schema';
+
+export interface BuiltInConfig {
+    accessToken?: string;
```

Comment:
This is not something Kap has.

Reply:
Okay, I was looking at the [description of OAuth](https://github.com/wulkano/kap/blob/master/docs/plugins.md#oauth). What's the right way to get the access token?

Reply:
It’s up to you to fetch it and store it.

Reply:
Ah, I see; it describes a recommended list of steps, not an enforced one, got it 🙂